test/icinga-perfdata.cpp

```diff
@@ -313,7 +313,16 @@ BOOST_AUTO_TEST_CASE(ignore_invalid_warn_crit_min_max)
 	BOOST_CHECK(pv->GetMin() == 3000);
 	BOOST_CHECK(pv->GetMax() == 4000);
 
-	BOOST_CHECK(pv->Format() == "test=123456");
+	BOOST_CHECK(pv->Format() == "test=123456;;;3000;4000");
+}
+
+BOOST_AUTO_TEST_CASE(empty_warn_crit_min_max)
+{
+	Array::Ptr pd = PluginUtility::SplitPerfdata("testA=5;;7;1;9 testB=5;7;;1;9 testC=5;;;1;9 testD=2m;;;1 testE=5;;7;;");
+	BOOST_CHECK(pd->GetLength() == 5);
+
+	String str = PluginUtility::FormatPerfdata(pd, true);
+	BOOST_CHECK(str == "testA=5;;7;1;9 testB=5;7;;1;9 testC=5;;;1;9 testD=120s;;;60 testE=5;;7");
```

Comment:
I know it's a bit awkward to request a change here since literally all the test cases in that file already do it that way, but it would be nice if you could use `BOOST_CHECK_EQUAL(actual, expect);` instead. Doing so would allow you to get the actual difference as part of the assertion's error messages without having to debug and re-run the tests to figure out what's actually going wrong. Here is what it could look like if the test case fails.
```cxx
... error: in "icinga_perfdata/empty_warn_crit_min_max": check str == "testA=5;;7;1;9 testB=5;7;;1;9 testC=5;;;1;9 testD=120s;;;60s testE=5;;7" has failed [testA=5;;7;1;9 testB=5;7;;1;9 testC=5;;;1;9 testD=120s;;;60 testE=5;;7 != testA=5;;7;1;9 testB=5;7;;1;9 testC=5;;;1;9 testD=120s;;;60s testE=5;;7]
```

Reply:
It is actually quite trivial to update all check in the file.